platform/debian/kanidmd/control

```diff
@@ -0,0 +1,14 @@
+Source: kanidmd
```

Comment:
I've always been uncertain about packaging the kanidmd binary on distros, given the docker container exists. The unixd and nsswitch pam, is unavoidable, but having only a "single way to run the server" is pretty attractive? Especially when there are so many ways to build it ...

Reply:
Some folks can't or won't run it on Docker, and I'm not sure I want to even go through the nightmares of *actually* submitting the packages to be blessed by the Debian/Ubuntu distributions, knowing the burden that that entails. (I might look at the PPA thing, but... not for a while)

This is more about enabling folks and providing dev builds since Github Actions are free, and adding `kanidmd` was a pretty simple addition.

Reply:
Yeah, that's fair. It's not me saying "remove it", but just airing a general thought process :)